idl/partition_checksum.idl.hh

```diff
@@ -76,4 +76,6 @@ struct get_combined_row_hash_response {
 
 enum class row_level_diff_detect_algorithm : uint8_t {
     send_full_set,
+    send_full_set_rpc_stream,
+};
 };
```

Comment:
Looks like extra } here

Reply:
And this patch misses the }. btw looks like github garbles my comments.

Reply:
fixed